reporter/src/main/kotlin/reporters/AmazonOssAttributionBuilderReporter.kt

```diff
@@ -104,8 +104,11 @@ class AmazonOssAttributionBuilderReporter : Reporter {
                 // URL passed into the Amazon Oss Attribution Builder needs to be reachable!
                 val pkgUrl = pkg.homepageUrl.takeUnless { it.isEmpty() } ?: "https://github.com/404"
 
-                val licensesWithCopyrights =
-                    input.ortResult.getDetectedLicensesWithCopyrights(pkg.id, input.packageConfigurationProvider)
+                val licenseInfo = input.licenseInfoResolver.resolveLicenseInfo(pkg.id)
+
+                val licensesWithCopyrights = licenseInfo.licenses
+                    .filter { LicenseSource.DETECTED in it.sources }
```

Comment:
I'm sorry if this is a stupid question, I'm not yet so familiar with the new API, but aren't we suppose to use a `LicenseView` for filtering things?

Reply:
No problem. I'm still new to this as well and wasn't even aware of the `LicenseView` option. I'll look into it, thanks!

Reply:
Done.